pkg/controller/perconaservermongodb/smart.go

```diff
@@ -24,6 +24,10 @@ import (
 func (r *ReconcilePerconaServerMongoDB) smartUpdate(cr *api.PerconaServerMongoDB, sfs *appsv1.StatefulSet,
 	replset *api.ReplsetSpec, secret *corev1.Secret) error {
 
+	if cr.Spec.Pause {
```

Comment:
I think the same here
```suggestion
	if replset.Size == 0 {
```